appdatabase/migrations/sql/0019_blocks_ranges_extra_data.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE blocks_ranges DROP COLUMN balance;
```

Comment:
that's not supported on sqlite, so I would just get rid of the `down` migration. 
Otherwise if you are really keen on having one you would have to create a temp table, copy over the data without the column, drop the initial one and move it over. Probably not worth as we don't run `down` migrations anyway.

Reply:
> Otherwise if you are really keen on having one
not really :D

I will remove it, thanks!